src/lib/components/works_for_you/notification.tsx

```diff
@@ -10,14 +10,15 @@ import colors from "../../../data/colors"
 
 interface Props extends RelayProps {}
 
-class Notification extends React.Component<Props, any> {
+export class Notification extends React.Component<Props, any> {
   render() {
     const notification = this.props.notification
 
     return (
       <View style={styles.container}>
         <View style={styles.header}>
-          <View>
+          { notification.image && <Image source={{uri: notification.image.resized.url}} style={styles.artistAvatar}/> }
```

Comment:
@katarinabatina the back button just shows up in the dev mode of emission - it won't show up once it's embedded in eigen.